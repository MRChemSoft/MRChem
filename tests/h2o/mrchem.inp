# vim:syntax=sh:

rel_prec = 1.0e-3                 # Overall relative precision

MRA {
  order = 6                       # Polynomial order
  min_scale = -5                  # Root scale defines world size 2^{-n}
  corner = [ -1, -1, -1 ]         # Translation of corner root box [x,y,z]
  boxes = [ 2, 2, 2 ]             # Number of root boxes [x,y,z]
}

Molecule {
$coords
O       0.0000  0.0000  -0.1250
H      -1.4375  0.0000   1.0250
H       1.4375  0.0000   1.0250
$end
}

WaveFunction {
  method = DFT                    # Wave function method (HF or DFT)
}

DFT {
  exact_exchange = 0.25
$functionals
PBEX    0.75
PBEC    1.00
$end
}

Properties {
  scf_energy = true               # Compute ground state energy
  dipole_moment = true            # Compute electrostatic dipole moment
}

SCF {
  orbital_thrs = 1.0e-2           # Convergence threshold in orbital residual
  initial_guess = GTO             # Type of initial guess: none, mw, gto
  kain = 3                        # Length of KAIN iterative history
  rotation = 5                    # Diagonalize Fock matrix every n'th iteration
  canonical = false               # Use canonical of localized orbitals
}

