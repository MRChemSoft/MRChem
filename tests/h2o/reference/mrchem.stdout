


************************************************************
***     __  __ ____   ____ _                             ***
***    |  \/  |  _ \ / ___| |__   ___ _ __ ___           ***
***    | |\/| | |_) | |   | '_ \ / _ \ '_ ` _ \          ***
***    | |  | |  _ <| |___| | | |  __/ | | | | |         ***
***    |_|  |_|_| \_\\____|_| |_|\___|_| |_| |_|         ***
***                                                      ***
***    VERSION 0.0.1 (rev. b47120b)                      ***
***                                                      ***
***    Stig Rune Jensen <stig.r.jensen@uit.no>           ***
***    Luca Frediani    <luca.frediani@uit.no>           ***
***    Peter Wind       <peter.wind@uit.no>              ***
***                                                      ***
************************************************************


+++ Serial execution


------------------------------------------------------------

 MRCPP version   : X.Y.Z
 Git revision    : 093397a

 Linear algebra  : EIGEN
 Parallelization : NONE

------------------------------------------------------------




============================================================
                  MultiResolution Analysis                  
------------------------------------------------------------
 polynomial order = 6
 polynomial type  = Interpolating
------------------------------------------------------------
 unit length      = 32.000000
 total boxes      = 8
 boxes            = [           2           2           2 ]
 lower bounds     = [  -32.000000  -32.000000  -32.000000 ]
 upper bounds     = [   32.000000   32.000000   32.000000 ]
 total length     = [   64.000000   64.000000   64.000000 ]
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     O           0.00000e+00   0.00000e+00  -1.25000e-01
  2     H          -1.43750e+00   0.00000e+00   1.02500e+00
  3     H           1.43750e+00   0.00000e+00   1.02500e+00
------------------------------------------------------------
 Center of mass:    4.62201e-18   0.00000e+00   3.68310e-03
============================================================


============================================================
                Setting up nuclear potential
------------------------------------------------------------
 Nr  Element         Charge        Precision     Smoothing 
------------------------------------------------------------
  1     O            8.00000e+00   1.00000e-03   5.10129e-04
  2     H            1.00000e+00   1.00000e-03   1.63241e-02
  3     H            1.00000e+00   1.00000e-03   1.63241e-02
============================================================


============================================================
                    Initialize DFT grid
------------------------------------------------------------
 Nr  Element                        nNodes          nPoints
------------------------------------------------------------
  0      O                             488          1174432
  1      H                             488          1174432
  2      H                             488          1174432
------------------------------------------------------------
                 Wall time: 1.15823e-03 sec 
============================================================


============================================================
                Setting up Gaussian-type MOs
------------------------------------------------------------
    n  Spin  Occ                           SquareNorm
------------------------------------------------------------
    0    p    2                       1.000000203953468e+00
    1    p    2                       1.000000065236651e+00
    2    p    2                       9.999999991988907e-01
    3    p    2                       1.000000008869738e+00
    4    p    2                       9.999999986945862e-01
------------------------------------------------------------
                 Wall time: 4.63529e+00 sec 
============================================================


============================================================
 OrbitalVector:   5 orbitals     5 occupied    10 electrons
------------------------------------------------------------
   n  RankID           Norm          Spin Occ      Error    
------------------------------------------------------------
   0     0       1.000000203953e+00    p   2    1.00000e+00
   1     0       1.000000065237e+00    p   2    1.00000e+00
   2     0       9.999999991989e-01    p   2    1.00000e+00
   3     0       1.000000008870e+00    p   2    1.00000e+00
   4     0       9.999999986946e-01    p   2    1.00000e+00
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                    1384      4.26665e-01
 Coulomb density                       456      3.23236e-01
 Coulomb potential                     328      2.28686e+00
 Coulomb grid size                     456          1251264
 Coulomb grid change                   128           351232
 XC total density                      488      3.37233e-01
 XC evaluate xcfun                    2440      3.36990e+00
 XC energy                             488      2.81430e-05
 XC potential                          488      6.53112e-02
 XC grid size                          488          1174432
 XC grid change                         96           230496
------------------------------------------------------------
                 Wall time: 6.85110e+00 sec 
============================================================


============================================================
                  Calculating Fock matrix
------------------------------------------------------------
 Kinetic part                         7.644412790000000e-01
 Potential part                       1.615104960000000e+00
------------------------------------------------------------
                 Wall time: 2.37962e+00 sec 
============================================================


============================================================
                    Localizing orbitals
------------------------------------------------------------
 Computing position matrices                    1.62716e+00
 Computing Foster-Boys matrix                   3.08569e-04
 Converged after iteration                               41
 Rotating orbitals                              8.05723e-02
------------------------------------------------------------
                 Wall time: 1.70812e+00 sec 
============================================================


============================================================
                  Setting up Fock operator
------------------------------------------------------------
 Precision                                      1.00000e-03
------------------------------------------------------------
 Nuclear potential                    1384      4.14486e-01
 Coulomb density                       648      4.22180e-01
 Coulomb potential                     328      2.33345e+00
 Coulomb grid size                     648          1778112
 Coulomb grid change                   -64          -175616
 XC total density                      648      3.98928e-01
 XC evaluate xcfun                    3240      4.37477e+00
 XC energy                             648      2.93750e-05
 XC potential                          648      8.80484e-02
 XC grid size                          648          1558592
 XC grid change                        -64          -153664
------------------------------------------------------------
                 Wall time: 8.08319e+00 sec 
============================================================


============================================================
                   Calculating SCF energy
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 1.74037e+00 sec 
============================================================


============================================================
                 Calculating dipole moment
------------------------------------------------------------
------------------------------------------------------------
                 Wall time: 1.60412e+00 sec 
============================================================


============================================================
                        Fock matrix
------------------------------------------------------------
-1.86631e+01  4.81611e-01  4.81611e-01 -9.18463e-01  9.18463e-01
 4.81612e-01 -5.98898e-01 -1.51844e-01  1.58105e-01 -1.58105e-01
 4.81612e-01 -1.51844e-01 -5.98898e-01  1.58105e-01 -1.58105e-01
-9.18465e-01  1.58104e-01  1.58105e-01 -3.94221e-01  1.71614e-01
 9.18465e-01 -1.58104e-01 -1.58105e-01  1.71614e-01 -3.94221e-01
============================================================


============================================================
                      Orbital energies
------------------------------------------------------------
    n  spin  occ                            epsilon  
------------------------------------------------------------
    0    p    2                      -1.878316105273624e+01
    1    p    2                      -8.928549720399752e-01
    2    p    2                      -4.470541097899429e-01
    3    p    2                      -3.036441057863551e-01
    4    p    2                      -2.226073437825361e-01
============================================================


============================================================
                          Molecule
------------------------------------------------------------
 Nr  Element             x             y             z      
------------------------------------------------------------
  1     O           0.00000e+00   0.00000e+00  -1.25000e-01
  2     H          -1.43750e+00   0.00000e+00   1.02500e+00
  3     H           1.43750e+00   0.00000e+00   1.02500e+00
------------------------------------------------------------
 Center of mass:    4.62201e-18   0.00000e+00   3.68310e-03
============================================================


                                                            
============================================================
                         SCF Energy                         
------------------------------------------------------------
                                                            
 Sum orbital energy:                 -4.129864316827009e+01
 Kinetic energy:                      7.584675423395471e+01
 E-N energy:                         -1.987273124965965e+02
 Coulomb energy:                      4.676227272199224e+01
 Exchange energy:                     0.000000000000000e+00
 X-C energy:                         -9.318932395562243e+00
 El. External field energy:           0.000000000000000e+00
 Nuc. External field energy:          0.000000000000000e+00
                                                            
------------------------------------------------------------
                                                            
 Electronic energy:                  -8.543721793621174e+01
 Nuclear energy:                      9.039235444235468e+00
                                                            
------------------------------------------------------------
                                                            
 Total energy       (au)             -7.639798249197626e+01
                    (kJ/mol)         -2.005828740014503e+05
                    (kcal/mol)       -4.794045742621132e+04
                    (eV)             -2.078894918632325e+03
                                                            
============================================================
                                                            

                                                            
============================================================
                         Dipole moment                      
------------------------------------------------------------
                                                            
 Length of vector:   (au)                  7.7928754277e-01
                     (Debye)               1.9807511747e+00
                                                            
------------------------------------------------------------
                                                            
   7.5414142551e-07   -8.7532485904e-12    7.7928754277e-01
                                                            
============================================================
                                                            



************************************************************
***                                                      ***
***                    Exiting MRChem                    ***
***                                                      ***
***               Wall time: 2.710673e+01                ***
***                                                      ***
************************************************************


